internal/services/mssql/mssql_server_resource.go

```diff
@@ -602,28 +620,32 @@ func expandSqlServerIdentity(input []interface{}) (*sql.ResourceIdentity, error)
 	return &out, nil
 }
 
-func flattenSqlServerIdentity(input *sql.ResourceIdentity) (*[]interface{}, error) {
+func flattenSqlServerIdentity(input *identity.LegacySystemAndUserAssignedMap) (*[]interface{}, error) {
```

Comment:
we should be able to replace usages of this function [with this one]
(https://github.com/hashicorp/go-azure-helpers/blob/6f888f20159ebfa9e44f565ef02bae5d438b0139/resourcemanager/identity/legacy_system_and_user_assigned_map.go#L96)?

Reply:
Fixed.